### PR TITLE
Griffin 2.0.0 dev

### PR DESCRIPTION
**Pull request by guoyuepeng**

### CODE REVIEW

**Comment by guoyuepeng** [comment]
hi everyone,

Could you have a check and work on this DAO service, I will work on Data Quality Workflow module.

@toyboxman 

Thanks.

File: griffin-bom/pom.xml [hunk 1]
```diff
@@ -37,12 +37,160 @@ under the License.
     <url>https://griffin.apache.org</url>
 
     <properties>
-
+        <spring-boot.version>2.7.3</spring-boot.version>
```
Comment (toyboxman) [inline griffin-bom/pom.xml:40]: Spring Boot 2.7x support expired in 2023.11, can you consider to use 3.2.x

**Comment by toyboxman** [comment]
All in all, the persistence skeleton looks good to me.

let's merge firstly and complement in following changes.

**Comment by toyboxman** [comment]
Again, what's JDK version do we prefer?  8, 11 , 17, or others, looks like spring boot latest support higher JDK.

Now in this PR, we simply include metric entity, yet workflow job still needs to be considered.

**Comment by guoyuepeng** [comment]
ok, I will merge it and we can improve it step by step...

**Pull request merged**